packages/escape-html/src/index.js

```diff
@@ -51,6 +51,17 @@ export function escapeLessThan( value ) {
 	return value.replace( /</g, '&lt;' );
 }
 
+/**
+ * Returns a string with greater-than sign replaced.
+ *
+ * @param {string} value Original string.
+ *
+ * @return {string} Escaped string.
+ */
+export function __unstableEscapeGreaterThan( value ) {
```

Comment:
Why are we even exposing it? If you're exposing it for tests, I'd recommend making a separate file and not export it in `index.js`.

Reply:
Related: https://github.com/WordPress/gutenberg/pull/9963#discussion_r237872160